jumpscale/clients/stellar/stellar.py

```diff
@@ -113,6 +114,15 @@ def _fund_transaction(self, transaction):
         resp.raise_for_status()
         return resp.json()
 
+    def _get_fund_transaction_conditions(self, asset):
+        resp = j.tools.http.post(self._get_url("CONDITIONS"))
+        resp.raise_for_status()
+        conditions = resp.json()
+        for condition in conditions:
+            if condition["asset"] == asset:
+                return condition
+        raise j.exceptions.Timeout(f"Conditions not found for asset: {asset}")
```

Comment:
Timeout?

Reply:
changed it to `NotFound` here https://github.com/threefoldtech/js-sdk/pull/3022/commits/29f545e3079918e8823baa8c99f166540afe53fd#diff-42adb1af219b45002b16f95f78c441b70a4c972e2affb5090f2f4d046ec7a999R124